scss/detail.scss

```
@@ -48,4 +48,19 @@
 .border-line {
   border-bottom: solid $earls-green;
   padding-bottom: 20px;
-} 
+}
+
+.badge {
+  position: relative;
+}
+
+.badge::after {
```

take advantage of sass for handling pseudoelements.
```suggestion
.badge {
  position: relative;
  
  &::after {}
}
```